views/layout.slim

```diff
@@ -38,12 +38,18 @@
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
     <link rel="stylesheet" href="/src/css/site.css">
+    <script src="/src/js/jobtracking.min.js"></script>
```

Comment:
Should probably have some kind of `<noscript>` tag for users that have JavaScript disabled.